conf/messages

```diff
@@ -19,22 +19,34 @@ error.maxLength = You must not enter more than {0} figures
 error.email= Enter a valid email address, like yourname@example.com
 
 ##Errors
+error.email.invalidchars=Business email address must only contain letters a to z, numbers, full stops, hyphens, underscores and @ signs
+error.email.maxlength=Business email address must be 132 characters or less
 error.postcode.invalid = Enter a valid postcode, for example AA1 1AA
+error.postcode.invalidchars=Postcode must only include letters a to z, numbers and spaces, like AA1 1AA
 error.postcode.blacklisted = You can''t use the postcode you''ve entered
 error.postcode.empty = Enter a postcode
+error.postcode.maxlength=Postcode must be 8 characters or less
 error.address.lines.empty = The address must have at least 1 line
 error.address.lines.invalid = The address must only include letters a to z, numbers, spaces, hyphens, commas, apostrophes, full stops and forward slashes
 error.address.lines.maxLength = You must not enter more than {0} figures
 error.utr.blank= Enter your business''s Unique Taxpayer Reference (UTR)
-error.utr.invalid.format = Unique Taxpayer Reference (UTR) must only include numbers and spaces
-error.utr.invalid.length = Unique Taxpayer Reference (UTR) must be 10 numbers
+error.utr.invalid = Unique Taxpayer Reference (UTR) must be 10 numbers
+error.sautr.blank=Enter your Self Assessment Unique Taxpayer Reference (UTR)
+error.sautr.invalid=Self Assessment Unique Taxpayer Reference (UTR) must be 10 numbers
+error.companyutr.blank=Enter your company Unique Taxpayer Reference (UTR)
+error.companyutr.invalid=Company Unique Taxpayer Reference (UTR) must be 10 numbers
+error.partnershiputr.blank=Enter your Partnership Unique Taxpayer Reference (UTR)
```

Comment:
I do not see any tests for this

Reply:
here it is https://github.com/hmrc/agent-subscription-frontend/pull/238/files#diff-a4a1a03183bc4c29884b210ecd1bf3b2R89